org.gnome.PowerStats.json

```diff
@@ -0,0 +1,68 @@
+{
+  "app-id": "org.gnome.PowerStats",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "command": "gnome-power-statistics",
+  "finish-args": [
```

Comment:
You probably should think about `dconf` access as well.
```
  "--filesystem=xdg-run/dconf",
  "--filesystem=~/.config/dconf:ro",
  "--talk-name=ca.desrt.dconf",
  "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```

See also:
http://docs.flatpak.org/en/latest/sandbox-permissions.html#dconf-access

Reply:
Added, thanks.